### ISSUE TITLE
Cannot chop down trees with GTCE axes.

### ISSUE THREAD

**Issue by Lgmrszd** [Gaelmare/dynamictreestfc#31]
Tested with mod versions (all latest at the moment of writing this issue):
TFC: 1.7.9.167
DT: 0.9.21
DTTFC: 0.9.15
GTCE: 1.10.5.582
Other DT trees are fine, I guess this has to do with "No Tree Punching" https://github.com/Gaelmare/dynamictreestfc/commit/5ccc97d2e510b2a916a63949ed202d087ce75a31, I may suggest to have an option to check not only tool class but also tool oredict (`craftingToolAxe`)

**Comment by agilmore2** [comment]
That's pretty strange, even TFC tools that don't extend vanilla tool classes have the tool class set. That's a forge tool classification that isn't related to inheritance or anything. I'm surprised that it's not doable. Let's discuss in #tng-addons.

**Comment by SpeeeDCraft** [comment]
Same problem

**Comment by Rongmario** [comment]
I was meaning to fix this in my own fork of GTCE. There's no real good way of fixing this on DTTFC's end so DTTFC shouldn't really explicitly make a case for it but urge GTCE devs currently to override `getToolClasses` in `MetaItem` or what not, since `getToolClasses` is made to be `ItemStack`-sensitive because of this very reason.

**Comment by SpeeeDCraft** [comment]
Please try it :)

**Comment by Gaelmare** [comment]
Rong isn't the gtce dev, so chances of a fix for this getting released are slim. It's up to GTCE to fix it.

**Comment by Rongmario** [comment]
Yeah I can PR to GTCE but not in any interest in doing so at the moment. Working on my own stuff.
